web/src/views/HomeView/StatsBar.vue

```diff
@@ -44,10 +44,6 @@ export default {
       dandisets: 0,
       users: 0,
       size: 0,
-      // Girder only
-      species: 0,
-      subjects: 0,
-      cells: 0,
```

Comment:
Not entirely sure these are supposed to go away. Need some help from @dchiquito to verify.

Reply:
These do need to go away. If you visit gui.dandiarchive.com you will notice that they are absent in the Django deployment.